src/InputWithLabel.js

```diff
@@ -0,0 +1,30 @@
+import React, { useRef, useEffect } from "react";
+
+export default function InputWithLabel({
+  todoTitle,
+  handleTitleChange,
+  children,
+}) {
+  const inputRef = useRef();
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
```

Comment:
This is the reason it wasn't autofocusing. Not your fault, there was a bug in the assignment. Delete this and it _should_ autofocus